stories/Speaker.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import { SpeakerProfile } from '../pages/speaker'
+
+storiesOf('SpeakerProfile', module).add('No talks', () => (
+  <SpeakerProfile name={'Donald Duck'}></SpeakerProfile>
+))
+storiesOf('SpeakerProfile', module).add('1 talk', () => (
+  <SpeakerProfile
+    name={'Donald Duck'}
+    presentations={[
+      {
+        event: { link: 'http://example.com' },
+        title: 'Example'
```

Comment:
Suggestion: "The Creation of Quack Pro"

Reply:
😂

https://en.wikipedia.org/wiki/Quackery ?

Reply:
I was more trying to make a Quake pun, but Quackery will do as well :laughing: